src/UploadedFile.php

```diff
@@ -16,6 +16,18 @@
 
 class UploadedFile implements UploadedFileInterface
 {
+    private static $errorMessages = [
```

Comment:
This is a good use case because it is immutable data. If it wasn't static it would be duplicated and it would increase the memory footprint by a lot.